src/main/java/net/dv8tion/jda/api/requests/GatewayIntent.java

```diff
@@ -377,7 +378,7 @@ public static EnumSet<GatewayIntent> fromEvents(@Nonnull Collection<Class<? exte
             else if (GenericUserUpdateEvent.class.isAssignableFrom(event) || GenericGuildMemberEvent.class.isAssignableFrom(event) || GuildMemberRemoveEvent.class.isAssignableFrom(event))
                 intents.add(GUILD_MEMBERS);
 
-            else if (GuildBanEvent.class.isAssignableFrom(event) || GuildUnbanEvent.class.isAssignableFrom(event))
+            else if (GuildBanEvent.class.isAssignableFrom(event) || GuildUnbanEvent.class.isAssignableFrom(event) || GuildAuditLogEntryCreateEvent.class.isAssignableFrom(event))
                 intents.add(GUILD_BANS);
```

Comment:
Guild Bans intent got renamed
```suggestion
                intents.add(GUILD_MODERATION);
```